### PR TITLE
EINT-33 update Readme

### PR DESCRIPTION
**Pull request by sahil-rokt**
### Background ###

Updated the Readme.md for ReactNative sdk.

Fixes [(issue)]

(https://rokt.atlassian.net/browse/EINT-33)

List out what has changed as a result of this PR.

### How Has This Been Tested? ###

Describe the tests that you ran to verify your changes. 

### Notes

Add any notes or extra information here that might be useful to the reviewer if applicable i.e. links to documentation/blogs or dashboards.

### Checklist: ###

- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have verified that CI completes successfully.
- [ ] All insignificant commits have been squashed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,35 @@
 
+
+
 # @rokt/react-native-sdk
 
-## Getting started
+## Resident Experts
+
+- Danial Motahari - danial.motahari@rokt.com
```
Comment (danial-rokt) [inline README.md:8]: I have not seen `Resident Experts` in other readmes. Do you have any samples?

Comment (DJingSeo) [reply README.md:8]: https://github.com/search?q=org%3AROKT+Resident+Experts&type=code

Comment (sahil-rokt) [reply README.md:8]: https://github.com/ROKT/rokt-sdk-web

File: README.md [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
+- Sahil Suri - sahil.suri@rokt.com
+
+## Overview
+
+This repository contains React Native SDK [Rokt.Widget](Rokt.Widget) as well as the Sample Application [RoktSampleApp](RoktSample) .
+
+The React Native SDK enables you to integrate Rokt into your React Native mobile apps to drive more value from—and for—your customers. The SDK is built to be lightweight, secure, and simple to integrate and maintain, resulting in minimal lift for your engineering team. The RoktSampleApp includes bare-minimum UI to demonstrate the usage of React Native SDK for our partners using the [Integration guide](##-integration-guide-for-demo-apps)
+
+## Development Environment
+Install ReactNative by running `npm install -g expo-cli`. The full description could be find here https://reactnative.dev/docs/environment-setup
```
Comment (danial-rokt) [inline README.md:18]: This installation is for expo which our SDK does not support it. Is it possible to change `npm install -g expo-cli`?

File: Rokt.Widget/README.md [hunk 3]
```diff
@@ -1,148 +1,42 @@
...
-`$ npm install @rokt/react-native-sdk --save`
+## Overview
+The React Native SDK enables you to integrate Rokt into your React Native mobile apps to drive more value from—and for—your customers. The SDK is built to be lightweight, secure, and simple to integrate and maintain, resulting in minimal lift for your engineering team.
+The SDK includes the common functionality in the ```*.js files``` whereas the platform specific code is written in the respective platform package example ```android or ios```.
+We use ```npm``` package manager to manage the dependencies for the project.
 
-### Mostly automatic installation
+| Environment | Build |
+| ----------- | :----- |
+| release |  [![CircleCI](https://circleci.com/bb/ROKT/rokt-sdk-ios/tree/master.svg?style=svg&circle-token=519d542734b554bf3484517306ccecddd243e78c)]
```
Comment (danial-rokt) [inline Rokt.Widget/README.md:15]: Is this one correct? It is pointing to rokt-sdk-ios repository

File: RoktSampleApp/README.md [hunk 4]
```diff
@@ -1,30 +1,56 @@
...
+## Overview
+The RoktSampleApp includes bare-minimum UI to demonstrate the usage of React Native SDK for our partners and testing
+The application includes the common functionality in the ```*.js files``` whereas the platform specific code is written in the respective platform package example ```sandroid or ios```.
+We use ```npm``` package manager to manage the dependencies for the project.
+
 
-## NPM install on Sample app
+| Environment | Build |
+| ----------- | :----- |
+| release |  [![CircleCI](https://circleci.com/bb/ROKT/rokt-sdk-ios/tree/master.svg?style=svg&circle-token=519d542734b554bf3484517306ccecddd243e78c)]
```
Comment (danial-rokt) [inline RoktSampleApp/README.md:18]: same here, rokt-sdk-ios !

File: RoktSampleApp/README.md [hunk 5]
```diff
@@ -1,30 +1,56 @@
...
-go to *../Rokt.Widget* directory and run `npm install`
-if there is any changes, run `npm pack`
+## Resident Experts
+
+- Danial Motahari - danial.motahari@rokt.com
+- Sahil Suri - sahil.suri@rokt.com
+
+## Overview
+The RoktSampleApp includes bare-minimum UI to demonstrate the usage of React Native SDK for our partners and testing
+The application includes the common functionality in the ```*.js files``` whereas the platform specific code is written in the respective platform package example ```sandroid or ios```.
```
Comment (danial-rokt) [inline RoktSampleApp/README.md:12]: typo `sandroid`

File: Rokt.Widget/README.md (hunk 3 above)
Comment (sahil-rokt) [reply Rokt.Widget/README.md]: yes fixed it, mistakenly missed changing

File: Rokt.Widget/README.md [hunk 6]
```diff
@@ -1,148 +1,42 @@
...
-      "firstname": "Jenny",
-      "lastname": "Smith",
-      "mobile": "(323) 867-5309",
-      "postcode": "90210",
-      "country": "US"
-}
-Rokt.execute("RoktEmbeddedExperience", attributes, placeholders,  () => console.log("Placement Loaded"));
-```
+To test the local changes, execute the ```$ npm pack ``` to create the SDK package name as ```rokt-react-native-sdk-X.X.X.tgz```. To change the version ```X.X.X``` , go to the ```package.json``` and modify the version field to new version. 
+Then we need to go RoktSampleApp project ```package.json``` to point to ``rokt-react-native-sdk-X.X.X.tgz``` file.
```
Comment (danial-rokt) [inline Rokt.Widget/README.md:34]: Do you want to mention that after `npm pack`, need to run `npm install` on RoktSampleApp folder and then they might need to run `pod install` on iOS folder?

File: RoktSampleApp/README.md [hunk 7]
```diff
@@ -1,30 +1,56 @@
...
+We use ```npm``` package manager to manage the dependencies for the project.
+
 
-## NPM install on Sample app
+| Environment | Build |
+| ----------- | :----- |
+| release |  [![CircleCI](https://dl.circleci.com/status-badge/img/gh/ROKT/rokt-sdk-react-native/tree/release-3.8.x.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/ROKT/rokt-sdk-react-native/tree/release-3.8.x)
```
Comment (danial-rokt) [inline RoktSampleApp/README.md:18]: This does not require to be changed, but we need to add this step to our release process to update this line when we release a new minor version

File: Rokt.Widget/README.md (hunk 6 above)
Comment (sahil-rokt) [reply Rokt.Widget/README.md:34]: In Step3, it is mentioned to ```npm install```,  will add pod install too

Comment (sahil-rokt) [reply Rokt.Widget/README.md:34]: that explanation is there in RoktSampleApp Readme.md about how to verify in sample app. Will add pod install step

File: README.md [hunk 8]
```diff
@@ -55,37 +85,73 @@ android {
...
+Below Configurations are required only if you are using Mac M1 machines:
+
+1. Make sure cocoa pods are installed using gem not brew(sudo gem install cocoapods).  
+If it is already installed using brew, use below commands to uninstall them  
+```
+brew uninstall cocoapods  
+brew uninstall --ignore-dependencies ruby
+```
+
+2. Install cocoapods with gem
```
Comment (danial-rokt) [inline README.md:104]: @DJingSeo on the readme of another repository, when we used technology like Cocoapods, you asked to have a description of it. 
In the ReactNative repository, we use terminologies like Cocoapods, Gradle, and gem, do you want to have the same standard here?
I just want to double-check with you not to have two different standards on Readmes.

File: RoktSampleApp/README.md (hunk 7 above)
Comment (sahil-rokt) [reply RoktSampleApp/README.md:19]: Added a note

File: README.md (hunk 8 above)
Comment (DJingSeo) [reply README.md:104]: Yes even 1-2 short sentences per the technology would be great!

**Comment by DJingSeo** [comment]
Build failed ....

**Comment by sahil-rokt** [comment]
https://github.com/ROKT/rokt-sdk-react-native/pull/63/files

Opened new PR

**Pull request closed without merging**